openapi/components/headers/X-RateLimit-Limit.yaml

```
@@ -0,0 +1,4 @@
+description: Overall rate limit tokens for this request.
```

suggestion: I'm not clear on what 'overall' means in this description

I think it's the total number?
```suggestion
description: Total number of rate limit tokens for this request.
```